features/gardener/exec.config

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
 
+# Install new docker.io version with new runc package
+wget --quiet http://ftp.de.debian.org/debian/pool/main/d/docker.io/docker.io_20.10.5+dfsg1-1+b1_amd64.deb -O /tmp/docker.io_20.10.5+dfsg1-1+b1_amd64.deb
+apt-get install -y -f /tmp/docker.io_20.10.5+dfsg1-1+b1_amd64.deb
+rm /tmp/docker.io_20.10.5+dfsg1-1+b1_amd64.deb
+
+wget --quiet http://ftp.de.debian.org/debian/pool/main/c/containerd/containerd_1.4.4~ds1-2_amd64.deb -O /tmp/containerd_1.4.4~ds1-2_amd64.deb
+apt-get install -y -f /tmp/containerd_1.4.4~ds1-2_amd64.deb
```

Comment:
Same as above, it can overwrite https://github.com/gardenlinux/gardenlinux/blob/main/features/gardener/file.include/etc/systemd/system/containerd.service.d/override.conf

Reply:
correct as well.